agent-server/nodejs/src/lib/BrowserAgentServer.js

```
@@ -1220,6 +1220,74 @@ export class BrowserAgentServer extends EventEmitter {
     }
   }
 
+  /**
+   * Execute JavaScript in a browser tab
+   * @param {string} tabId - Tab ID (target ID)
```

ESLint

[error] There must be no hyphen before @param description.

(jsdoc/require-hyphen-before-param-description)